static/css/calendar-icon.less

```diff
@@ -0,0 +1,92 @@
+/* ==========================================================================
+   cfgov-refresh
+   calendar-icon
+   ========================================================================== */
+
+/* topdoc
+  name: Calendar Icon
+  family: cfgov-calendar-icon
+  notes:
+    - "A square-ish calendar icon with a three-letter month name and date."
+  patterns:
+    - name: Calendar Icon
+      markup: |
+        <time class="calendar-icon" data-month="Jan" data-day="21" datetime="2015-01-21">
+          <span class="u-visually-hidden">Jan 21, 2015</span>
+        </time>
+        denotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          time.calendar-icon
+            span.u-visually-hidden
+  tags:
+    - cfgov-calendar-icon
+*/
+
+
+/*  Mixin to generate a calendar icon.
+    @month [String] The month (3-letter abbreviation, typically).
+    @day [Number] The day in the month.
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/3b149a0f5def00bf854148daeb61fe03a2e26f4d